operator/pkg/central/extensions/reconcile_admin_password.go

```diff
@@ -184,7 +194,7 @@ func (r *reconcileAdminPasswordExtensionRun) generateHtpasswdSecretData() (types
 func (r *reconcileAdminPasswordExtensionRun) viewPasswordInstructionsMessage() string {
 	return fmt.Sprintf(
 		"A password for the 'admin' user has been automatically generated and stored in the %q entry of the central-htpasswd secret.\n"+
-			"To view the password, run\n"+
+			"To view the password click the secret reference below or run\n"+
```

Comment:
`click the secret reference below` is very console-centric and won't make sense to someone viewing the CR with something like `kubectl get centrals`. How about this instead:
```suggestion
			"To view the password see the secret reference field or run\n"+
```